src/js/src/components/SearchSingleItemComponents/SearchMasonryImage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="columnImageContainer">
+    <div class="number">
+      #{{ row + number * rowNumber }}
+    </div>
+    <img 
+      loading="lazy"
+      :src="result.downloadUrl"
+      @click="toggleFullImage(row + number * rowNumber)">
+    <search-single-item-focus-image v-if="showFullImage === row + number * rowNumber"
```

Comment:
Out of curiosity - why is this show/hide toggling of full image based on _row + number * rowNumber_ ?

Reply:
Good catch. This is not needed when the the parent only has one image to show. The focus image template is also used in t the images shown in a post, which can contain alot of images all using the same single template - so in that context i have to keep track of which image is shown. I don't have to do that here, so I've changed it.